node/tests/SharedTests.ts

```diff
@@ -873,6 +873,74 @@ export function runBaseTests<Context>(config: {
         config.timeout,
     );
 
+    it.each([ProtocolVersion.RESP2, ProtocolVersion.RESP3])(
+        `lset test_%p`,
+        async (protocol) => {
+            await runTest(async (client: BaseClient) => {
+                const key = "test_key";
+                const nonExistingKey = "nonexisting";
```

Comment:
Should probably use uuidv4 to avoid key name conflicts between tests
```suggestion
                const key = uuidv4();
                const nonExistingKey = uuidv4();
```